addons/web/static/src/views/fields/field.js

```diff
@@ -159,15 +157,16 @@ export class Field extends Component {
                 if (record.selected && record.model.multiEdit) {
                     return;
                 }
+                const isInEdition = rootRecord ? rootRecord.isInEdition : record.isInEdition;
```

Comment:
move rootRecord definition here?
```suggestion
                const isInEdition = record.model.root instanceof record.constructor ?
                    record.model.root.isInEdition :
                    record.isInEdition;
```